common/utils.go

```diff
@@ -124,3 +126,48 @@ func MapStringStructToSlice(m map[string]struct{}) []string {
 	}
 	return s
 }
+
+// GetNumPossibleCPUs returns a total number of possible CPUS, i.e. CPUs that
+// have been allocated resources and can be brought online if they are present.
+// The number is retrieved by parsing /sys/device/system/cpu/possible.
+//
+// See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/linux/cpumask.h?h=v4.19#n50
+// for more details.
+func GetNumPossibleCPUs(log *logrus.Entry) int {
+	f, err := os.Open(PossibleCPUSysfsPath)
+	if err != nil {
+		log.WithError(err).Errorf("unable to open %q", PossibleCPUSysfsPath)
```

Comment:
maybe we should `return 0` in case of error.

Reply:
Fixed. I guess it used to fail a bit later with `unable to read [...] to get CPU count`. Best to fail early.